package.json

```diff
@@ -87,21 +87,6 @@
       "config": "./tooling/cz-config.js"
     }
   },
-  "lint-staged": {
```

Comment:
I assume these settings aren't needed with the new version.

Reply:
These now live in `lint-staged.config`